changelog/4391.feature.rst

```diff
@@ -0,0 +1 @@
+Improved time counter used to compute test durations. ``<CallInfo>.start`` is now retro-computed as ``<CallInfo>.stop - duration``, where ``duration`` is counted using ``time.perf_counter()``.
```

Comment:
I believe this should be `4391.improvement.rst` instead of `4391.feature.rst`, I believe. 👍

Reply:
Great proposal indeed.